main.js

```diff
@@ -1,26 +1,27 @@
 
-const electron      = require('electron');
```

Comment:
aww, my formatter killed this.. Can someone check this out?

Reply:
what do you mean @kewde ?

Reply:
@pciavald I liked the previous formatting, where the '=' are all aligned vertically.